AI_Graphics/AI_Graphics/Inspector/LightingInspector.cs

```diff
@@ -45,32 +45,37 @@ internal static void Draw(LightingSettings lightingSettings, SkyboxManager skybo
 
```

Comment:
with many cubemaps, the selection needs to be restricted else it makes the configuration panels really cramped. can you change the cubemapscrollview to something like this?
`cubeMapScrollView = GUILayout.BeginScrollView(cubeMapScrollView, GUILayout.MaxHeight(300));`

Reply:
Fixed